static/scss/answers/common/mixins.scss

```diff
@@ -135,7 +135,7 @@
     margin-inline-end: var(--yxt-base-spacing);
   }
 
-  h1 
+  h1, .js-yxt-rtfValue > *:first-child // The first item should also get h1 styling
```

Comment:
oops, I need to make sure this only applies to rich text featured snippets. Thanks percy!